src/lib/proxy-server.ts

```diff
@@ -97,7 +125,10 @@ export async function startProxyServer(): Promise< boolean > {
 
 		return true;
 	} catch ( error ) {
-		if ( error instanceof Error && 'code' in error && error.code === 'EADDRINUSE' ) {
+		if (
+			( error instanceof Error && isErrnoException( error ) && error.code === 'EADDRINUSE' ) ||
```

Comment:
```suggestion
			( isErrnoException( error ) && error.code === 'EADDRINUSE' ) ||
```
Sorry for the nitpicking, but `isErrnoException` already checks if `value instanceof Error`